### ISSUE TITLE
nsqadmin: nodes says zero without lookupd configured

### ISSUE THREAD

**Issue by Dieterbe** [bitly/nsq#627]
on the hosts page, it says 0 hosts
![one](https://cloud.githubusercontent.com/assets/20774/9191686/89c1b3b2-4004-11e5-8064-b6ba0a49bc2e.png)
but on the stream page, it finds the stream running on my nsqd instance, in fact when you click on it.. see below
![two](https://cloud.githubusercontent.com/assets/20774/9191688/8c2f0c9e-4004-11e5-844e-4756d06b922f.png)
it takes you to this page. but the main hosts page still says 0.
![three](https://cloud.githubusercontent.com/assets/20774/9191704/bd55402c-4004-11e5-9e7f-7a207aa11273.png)

runs in docker
```
b3e02f40d6fe        nsqio/nsq:latest                 "/nsqadmin --nsqd-ht   3 minutes ago       Up 3 minutes        4150-4151/tcp, 4160-4161/tcp, 4170/tcp, 0.0.0.0:4171->4171/tcp                                                     raintankdocker_nsqadmin_1                
407f61435d42        nsqio/nsq:latest                 "/nsqd"                3 minutes ago       Up 3 minutes        4151/tcp, 4160-4161/tcp, 0.0.0.0:4150->4150/tcp, 4170-4171/tcp                                                     raintankdocker_nsqd_1 
```

nsqadmin show it's querying nsqd just fine:
```
2015/08/11 06:29:30 NSQD: querying http://nsqd:4151/stats?format=json
```

**Comment by Dieterbe** [comment]
oh this is 0.3.5 btw since it's the docker images.

**Comment by jehiah** [comment]
While nsqadmin changed quite a bit in #323 from the version you are testing, this functionality is the same; the 'nodes' page currently lists nodes that are registered in nsqlookupd,  but it's not obvious from the page, and there probably isn't a good reason for it to stay that way. 

It should be straight forward to have it also list the nsqd instances configured on the command line if not using nsqlookupd [here](https://github.com/bitly/nsq/blob/master/nsqadmin/http.go#L238). Interested in taking a pass at that?

**Comment by Dieterbe** [comment]
don't think i'll have time soon, sorry.

**Comment by mreiferson** [comment]
updated title

**Comment by mreiferson** [comment]
fixed in #642
